src/server.js

```diff
@@ -17,7 +17,7 @@ const startServer = async (portNumber) => {
     app.use(express.json())
 
     app.post('/api/notifications', resetPassword)
-
+    app.patch('/api/users/:id',updateDBPassword)
```

Comment:
This needs to be tested too. Doesn't it?

Reply:
yea i changed my original test to meet carlos' code. i was thinking his code in the server.js fire was right...😕